src/app/workflow-app/workflow-app.component.ts

```diff
@@ -107,6 +112,31 @@ export class WorkflowAppComponent implements AfterContentInit {
     }
   }
 
+  @HostListener('document:mousedown', ['$event'])
+  onMouseDown($event: MouseEvent): void {
+    if (($event.srcElement.className == "close-button") && 
```

Comment:
Please try to find some other way: your code will break with any number of otherwise innocuous changes to the template (e.g., changes to classes and also translation).

Using text, DOM, or css to support logic is almost never correct. You can use @ViewChild, and/or you can have the onclick's of different elements call a common function either directly or indirectly.